src/index.js

```diff
@@ -319,5 +357,56 @@ function populateCustomerDash(customerDashInfo) {
   bookingsList.innerHTML = generateBookingsList(customerDashInfo.userBookings); 
 }
 
-// it should also be used to calculate & display their total spent(functions already exist for this ?)
-//   it should also be used to generate a list of their bookings(again, functions already exist for this; may need to be made more dynamic)
+function getRoomsAvailableOnDate() {
+  event.preventDefault();
```

Comment:
This may be why you cannot use the calendar! I think I ran into a similar issue. I commented this line out and it did let me see the drop down calendar, but it only showed me the results for a second and it took me back to the login screen.

Reply:
Hmm I guess the default behavior of date is both refreshing the page & allowing the date picker to be used? I tried googling this and couldn't find any alternative that would allow me not to use event.preventDefault(). If you find anything, let me know!

Reply:
The only difference I spotted is currently I am not using event.preventDefault() only on that particular click event and the calendar is dropping down, letting me select a date and click search. The page then updates to display the available rooms on that date. I even tried commenting out lines 361 and 368 and it will still take me back to the login page upon click of 'Search'. I am stumped!